backend/src/db/queries.ts

```diff
@@ -12,7 +12,7 @@ import { logger } from '../utils/logger';
  *
  * @returns {unknown[]} https://node-postgres.com/apis/result. Currently returns result.rows.
  */
-const executeQuery = async (
+export const executeQuery = async (
```

Comment:
We probably don't want execureQuery to be used directly outside this file

Reply:
I don't see a point for creating custom methods in queries file for single jest tests.
Personally, I understand the tests better if I see the actual database queries rather than method names that do the same thing.

Overall, the logic in the tests would be the same, but we just would need to create much more functions in queries file without much benefit.

Reply:
I understand your point with the tests but currently even the tests don't use this function directly. Therefore this only opens a possibility for someone to use this function incorrectly somewhere. As long as this function is kept private inside this file and the other functions are used instead we have better control over the queries sent to the database.

For example we can easily ensure that the queries are allways constructed correctly and SQLi attacks are not possible. If this function is exported it can be accidentally used in a way that makes such attacks possible.

This file also provides a clean interface for accessing the data base. Ideally that interface could be kept unchanged even if there were drastic changes in the database solution just by modifying the contents of this file. This advantage will be also lost if we expose the lower level functions.